src/store/propertiesSlice.ts

```diff
@@ -32,3 +38,13 @@ export const fetchPostsTypes = (): AppThunkType => async (dispatch) => {
 
   dispatch(loadPostsTypes(postTypes));
 };
+
+export const fetchPostsTags = (): AppThunkType => async (dispatch) => {
+  const response = await getTag({
+    params: {
+      value: 'а',
```

Comment:
a ? :)

Reply:
i thought i had to make a request for random tags, so i decided to ask for 'a' ))
will explain it tomorrow